src/__tests__/utils/password.test.ts

```diff
@@ -0,0 +1,31 @@
+import { checkHash, hashFunction } from '../../utils/password';
+import { getRandomString } from '../../__mocks__/pages/api/getRandom';
+
+describe('testing the hashFunction', () => {
+  it('should compare the secret hashed and without', async () => {
+    expect(await hashFunction('secret_hi')).not.toBe('secret_hi');
+    expect(await hashFunction('secret_hi')).not.toBe(
+      await hashFunction('secret_hi')
+    );
+  });
+
+  it('should compare the length of 2 hashed passwords', async () => {
+    //add 2 random length passwords: 1-10
+    const len1 = Math.floor(Math.random() * 10 + 1);
+    const len2 = Math.floor(Math.random() * 10 + 1);
```

Comment:
Make another mock called randomNumber and assign it instead of hardcoding number 10